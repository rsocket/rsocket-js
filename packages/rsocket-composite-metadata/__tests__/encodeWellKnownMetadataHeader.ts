import {
  encodeWellKnownMetadataHeader,
  WellKnownMimeType,
} from "rsocket-composite-metadata";

describe("encodeWellKnownMetadataHeader", () => {
  it("encodes the header as per spec", () => {
    const header = encodeWellKnownMetadataHeader(
      WellKnownMimeType.APPLICATION_JSON.identifier,
      WellKnownMimeType.APPLICATION_JSON.toString().length
    );
    const actual = header.toJSON().data;
    expect(actual).toMatchSnapshot();
  });
});
