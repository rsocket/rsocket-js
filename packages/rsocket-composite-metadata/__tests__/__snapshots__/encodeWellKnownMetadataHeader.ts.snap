// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`encodeWellKnownMetadataHeader encodes the header as per spec 1`] = `
Array [
  133,
  0,
  0,
  16,
]
`;
